arch/arm/src/armv7-a/arm_syscall.c

```diff
@@ -35,10 +35,11 @@
 #include <nuttx/sched.h>
 #include <nuttx/addrenv.h>
 
-#include "signal/signal.h"
-#include "arm.h"
 #include "addrenv.h"
+#include "arm.h"
 #include "arm_internal.h"
+#include "group/group.h"
+#include "signal/signal.h"
```

Comment:
don't need include signal/signal.h and addrenv.h

Reply:
Depends by: nxsig_unmask_pendingsignal